api/types/resource.go

```diff
@@ -469,6 +469,23 @@ func MatchLabels(resource ResourceWithLabels, labels map[string]string) bool {
 	return true
 }
 
+// MatchKinds takes an array of strings that represent a Kind and
+// returns true if the resource's kind matches any item in the given array.
+func MatchKinds(resource ResourceWithLabels, kinds []string) bool {
+	if len(kinds) == 0 {
+		return true
+	}
```

Comment:
This condition is redundant.

Reply:
yeah it seems like all the `Match` methods in this area have that redundant conditional. Perhaps its used as a safety check due to the public nature of the method? Should be ok to stay just in case?

Reply:
If it's consistent with the code around it, then yes, let's leave it.